docs/UserGuide.md

```diff
@@ -136,6 +136,36 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd patient in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st patient in the results of the `find` command.
 
+### Adding an appointment: `assign`
+
+Assign the specified patient into the specified appointment date and time.
+
+Format: `assign INDEX d/DATE t/TIME`
+
+* Puts the person at the specified INDEX into an appointment time slot.
+* The INDEX refers to the index number indicated in the patient list.
+* The IC **must be a positive integer** 1, 2, 3, …​
```

Comment:
Hmm was wondering if IC here will now refer to INDEX instead as we adopted the new command syntax?

Reply:
Oh I forgot to change it to INDEX. Thanks for spotting it 👍